core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kubernetes/upgrade-kubeadm-config.yml

```diff
@@ -1,4 +1,12 @@
 ---
-- name: upgrade-kubeadm-config | Save kubeadm-config ConfigMap to file
-  shell: >-
-    kubeadm config view > /etc/kubeadm/kubeadm-config.yml
+- name: upgrade-kubeadm-config | Check if /etc/kubeadm/kubeadm-config.yml exists
```

Comment:
I think we always want to have this file (it is also some kind of backup) so we could keep the old version of this file - it's simpler.